MacOS build failing

# Description

```
go: downloading github.com/sirupsen/logrus v1.8.1
go: downloading golang.org/x/sys v0.0.0-20191026070338-33540a1f6037
# golang.org/x/sys/unix
Error: ../../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191026070338-33540a1f6037/unix/syscall_darwin.1_13.go:25:3: //go:linkname must refer to declared function or variable
Error: ../../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191026070338-33540a1f6037/unix/zsyscall_darwin_amd64.1_13.go:27:3: //go:linkname must refer to declared function or variable
Error: ../../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191026070338-33540a1f6037/unix/zsyscall_darwin_amd64.1_13.go:40:3: //go:linkname must refer to declared function or variable
Error: ../../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191026070338-33540a1f6037/unix/zsyscall_darwin_amd64.go:28:3: //go:linkname must refer to declared function or variable
Error: ../../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191026070338-33540a1f6037/unix/zsyscall_darwin_amd64.go:43:3: //go:linkname must refer to declared function or variable
Error: ../../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191026070338-33540a1f6037/unix/zsyscall_darwin_amd64.go:59:3: //go:linkname must refer to declared function or variable
Error: ../../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191026070338-33540a1f6037/unix/zsyscall_darwin_amd64.go:75:3: //go:linkname must refer to declared function or variable
Error: ../../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191026070338-33540a1f6037/unix/zsyscall_darwin_amd64.go:90:3: //go:linkname must refer to declared function or variable
Error: ../../../go/pkg/mod/golang.org/x/sys@v0.0.0-20191026070338-33540a1f6037/unix/zsyscall_darwin_amd64.go:105:3: //go:linkname must refer to declared function or variable
```

# Steps to reproduce?

Provide specific examples to reproduce the issue. Please also list any relevant details for your configuration.

- [x] `make build`

**Run Configuration**:
- Go Proxy version: `1.0.3`
- Go version: `1.18`
- Operating system: `macOS`
- Processor architecture: `amd64`, `arm64`

### Actual behavior

MacOS build is failing.

### Expected behavior

MacOS build will succeed.

# Checklist

- [x] I have read the [CONTRIBUTING.md](https://github.com/livesport-tv/goproxy/blob/master/CONTRIBUTING.md).
- [x] I have checked the [Q&A](https://github.com/livesport-tv/goproxy/discussions/categories/q-a) for a list of common questions and problems.
- [x] I have performed a search in [issues](https://github.com/livesport-tv/goproxy/issues) and [discussions](https://github.com/livesport-tv/goproxy/discussions) to see if the problem has already been reported.
- [x] The reported problem is a bug and not the expected behavior.

@schenkova @bafrnec

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,14 @@
 
 ## [Unreleased]
 
+## [1.0.4] - 2022-03-17
+### Changed
+- Updated:
+  - `golang.org/x/sys`  to `v0.0.0-20220317061510-51cd9980dadf` (indirect dependency)
+
+### Fixed
+- MacOS build by updating `golang.org/x/sys`.
+
 ## [1.0.3] - 2022-03-17
 ### Changed
 - Updated to Go 1.18.
@@ -23,7 +31,8 @@
 ### Added
 - First release of Go Proxy.
 
-[Unreleased]: https://github.com/livesport-tv/goproxy/compare/v1.0.3...master
+[Unreleased]: https://github.com/livesport-tv/goproxy/compare/v1.0.4...master
+[1.0.4]: https://github.com/livesport-tv/goproxy/compare/v1.0.3...v1.0.4
 [1.0.3]: https://github.com/livesport-tv/goproxy/compare/v1.0.2...v1.0.3
 [1.0.2]: https://github.com/livesport-tv/goproxy/compare/v1.0.1...v1.0.2
 [1.0.1]: https://github.com/livesport-tv/goproxy/compare/v1.0.0...v1.0.1

--- a/go.mod
+++ b/go.mod
@@ -10,6 +10,6 @@ require (
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	golang.org/x/sys v0.0.0-20191026070338-33540a1f6037 // indirect
+	golang.org/x/sys v0.0.0-20220317061510-51cd9980dadf // indirect
 	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
 )

--- a/openapi.yaml
+++ b/openapi.yaml
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 info:
   description: "Go proxy with monorepo support (more projects with `go.mod` and different versions in the same GIT repository)."
-  version: "1.0.3"
+  version: "1.0.4"
   title: "Go Proxy"
   contact:
     name: "maintainer"

